cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -562,6 +584,21 @@ Entities that match the specified query and entity type are returned in the body
    * - ``showHidden``
      - Metadata search hides entities whose name starts with "_" (underscore) from search results. 
        Set this to ``true`` to include these hidden entities in search results. Default is ``false``.
+   * - ``sort``
+     - The sorting order for the ``results`` being returned. Default is to sort search results as a function of
+     relative weights for the specified search query. Specify the sort order by specifying the field name followed
+     by ``ASC`` or ``DESC``, with a '+' separating the two words. Example: ``&sort=creation-time+asc`
+   * - ``offset``
+     - The number of search results to skip before including them in the ``results``. Default is ``0``.
+   * - ``limit``
+     - The number of metadata search entities to return in the ``results``. By default, there is no limit.
+   * - ``numCursors``
+     - An upper limit on the number of cursors to return after the the ``limit`` parameter is applied to
```

Comment:
Only used when the search query is ``*``. 

How about "Determines the number of chunks of search results of size ``limit`` to fetch after the first chunk of size ``limit``. This parameter can be used to roughly estimate the total number of results that match the search query. Only used when the search query is ``*``"